From 4776cf82a9367ff51883af243a219d45da35d3b5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Timo=20Ter=C3=A4s?= <timo.teras@iki.fi>
Date: Wed, 23 Apr 2014 09:11:46 +0300
Subject: [PATCH] default to localtime timezone if TZ is undefined

the rest of the logic fallsback to GMT if it does not exist
(since musl commit 0f2315b4af1c58cbfb7c7f9da69b495cd146cc18)
---
 src/time/__tz.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/time/__tz.c b/src/time/__tz.c
index 6d7173c..4184b60 100644
--- a/src/time/__tz.c
+++ b/src/time/__tz.c
@@ -128,7 +128,7 @@ static void do_tzset()
 		"/usr/share/zoneinfo/\0/share/zoneinfo/\0/etc/zoneinfo/\0";
 
 	s = getenv("TZ");
-	if (!s || !*s) s = __gmt;
+	if (!s || !*s) s = ":localtime";
 
 	if (old_tz && !strcmp(s, old_tz)) return;
 
-- 
1.9.2

